core/domain/prod_validation_jobs_one_off.py

```diff
@@ -584,7 +595,11 @@ def validate(cls, item):
         super(BaseCommitLogEntryModelValidator, cls).validate(item)
 
         cls._validate_post_commit_status(item)
-        cls._validate_post_commit_is_private(item)
+
+        if 'question' in item.id or 'skill' in item.id:
```

Comment:
Can you make this check more explicit (by parsing or extracting the relevant part of item.id)? It may happen rarely but it's possible that a random exploration ID contains the letters 's' 'k' 'i' 'l' 'l' in order.

Reply:
Done